sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQuery.scala

```diff
@@ -161,4 +162,9 @@ trait StreamingQuery {
    * @since 2.0.0
    */
   def explain(extended: Boolean): Unit
+
+  /**
+   * Returns the total input records of this streaming query.
```

Comment:
The scaladoc comment should be more general one, as QuerySummary is designed to be used for general purposes.

Reply:
`QuerySummary` has been removed and all these UI data will be tracked in StreamingQueryListener. Mark this resolved.